omou-frontend/src/reducers/rootReducer.js

```diff
@@ -3,12 +3,14 @@ import Registration from "./registrationReducer";
 import auth from "./authReducer";
 import Course from "./courseReducer";
 import Users from "./usersReducer";
+import Calender from "./calenderReducer"
 
 const rootReducer = combineReducers({
     Registration,
     auth,
     Course,
-    Users
+    Users,
+    Calender
```

Comment:
always remember to leave a comma even after the last item in the list - it helps the next person working on this